releases/release-1.30/README.md

```diff
@@ -0,0 +1,121 @@
+- **Tuesday 19th - Friday 22nd March 2024**: Week 11 - [KubeCon EU](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/)
+- **[01:00 UTC Wednesday 20th March 2024 / 18:00 PDT Tuesday 19th March 2024](https://everytimezone.com/s/b708fa7a)**: Week 11 — [Test Freeze](../release_phases.md#test-freeze)
+- **Tuesday 26th March 2024**: Week 12 — [Docs Freeze](../release_phases.md#docs-freeze)
+- **Wednesday 17th April 2024**: Week 15 — Kubernetes v1.30.0 released
+
+## Timeline
+
+| **What**                                                                               | **Who**                       | **When**                                                                                                              | **Week** | **CI Signal**                                          |
+|----------------------------------------------------------------------------------------|-------------------------------|-----------------------------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------|
+| Start of Release Cycle                                                                 | Lead                          | Monday 8th January 2024                                                                                             | week 1   | [master-blocking]                                      |
+| Start Enhancements Tracking                                                            | Enhancements Lead             | Monday 8th January 2024                                                                                             | week 1   |                                                        |
+| Schedule finalized                                                                     | Lead                          | Friday 12th January 2024                                                                                             | week 1   |                                                        |
+| Team finalized                                                                         | Lead                          | Friday 12th January 2024                                                                                             | week 1   |                                                        |
+| Start Release Notes Draft                                                              | Release Notes Lead            | Tuesday 23th January 2024                                                                                           | week 3   |                                                        |
+| 1.30.0-alpha.1 released                                                                | Branch Manager                | Monday 29th January 2024                                                                                            | Week 4   |                                                        |
```

Comment:
We tend to avoid Monday/Friday releases unless necessary because of availability of release managers and timezones. Would it work to move alpha.1 to Tuesday, Jan 30?